.aiidashellrc

```diff
@@ -0,0 +1,15 @@
+alias create bk backward
```

Comment:
I would be tempted to remove as much as possible custom aliases, and let instead the user define them, unless we are 100% confident that eveyone wants them.
We could autoload the rc file by expecting to find it in the AiiDA profile directory, similar to what I did in my PR for the history file? I think that is the right place to put it?

(Edit: I saw below this is just intended as an example; then it's OK - but I would still think to just a few defaults that we are kind of convinced we want almost everybody to use)